aspnet/Minitwit.Test/Repositories/MessageRepositoryTest.cs

```diff
@@ -0,0 +1,83 @@
+﻿using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+using Minitwit.Models.Context;
+using Minitwit.Models.Entity;
+using Minitwit.Repositories;
+using Minitwit.Test.Context;
+using System;
+using System.Threading.Tasks;
+using Xunit;
+using System.Linq;
+
+namespace Minitwit.Test.Repositories
+{
+    public class MessageRepositoryTest : IDisposable
+    {
+        private readonly SqliteConnection _connection;
+        private readonly MinitwitContext _context;
+        private readonly MessageRepository _repository;
+        private readonly UserRepository _userRepository;
+
+        public MessageRepositoryTest()
+        {
+            _connection = new SqliteConnection("DataSource=:memory:");
+            _connection.Open();
+            var builder = new DbContextOptionsBuilder<MinitwitContext>().UseSqlite(_connection);
+            _context = new MinitwitTestContext(builder.Options);
+            _context.Database.EnsureCreated();
+            _repository = new MessageRepository(_context);
+            _userRepository = new UserRepository(_context);
+        }
+
+        [Fact]
+        public async Task Test_get_private_timeline()
```

Comment:
I would have prefferred the WHEN_x_GIVEN_y_THEN_z pattern

Reply:
Changed naming convention